packages/onboard-react/src/components/HardwareWalletSteps/HardwareWalletSteps.tsx

```diff
@@ -0,0 +1,85 @@
+import { FC, useCallback, useState } from "react";
+import {
+  Account,
+  selectAccount,
+  selectAccountOptions,
+} from "@sovryn/onboard-hw-common";
+import { DerivationPathForm } from "./DerivationPathForm";
+import { AddressListTable } from "./AddressListTable";
+
+export enum HardwareWalletStep {
+  derivationPathForm,
+  addressList,
+}
+
+type HardwareWalletStepsProps = {
+  step: HardwareWalletStep;
+  onStepChanged: (step: HardwareWalletStep) => void;
+};
+
+export const HardwareWalletSteps: FC<HardwareWalletStepsProps> = ({
+  step,
+  onStepChanged,
+}) => {
+  const [chainId] = useState(selectAccountOptions.chains[0].id);
+  const [asset] = useState(selectAccountOptions.assets[0]);
```

Comment:
Extract these variables outside of the component.

Reply:
It needs to be in scope of component, to make sure `selectAccountOptions` actually defined when component is loaded.